Cargo.toml

```diff
@@ -11,13 +11,14 @@ edition = "2018"
 
 [dependencies]
 actix = "0.10"
+anyhow = "1.0.40"
```

Comment:
Here and everywhere else, it should be enough to keep dependencies labels as "x.y" to avoid a lot z-updating activity (unless we know we strictly depend on some bugfix in a given a z-version).
The `Cargo.lock` in the this repo is the one meant to keep track of specific "x.y.z" pinnings.

Reply:
ohh that makes sense. thanks!